devtools/jsonapi/json_client.go

```diff
@@ -148,14 +179,18 @@ func SendTransaction(transferJson *Transaction, keyPair *keys.Ed25519KeyPair, se
 }
 
 func CallMethod(transferJson *Transaction, serverUrl string, logVerbose bool) (*CallMethodOutput, error) {
-	tx := ConvertTransaction(transferJson)
+	tx, err := ConvertTransaction(transferJson)
+	if err != nil { // The JSON we got is probably invalid so we exit
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
Please handle errors on higher level, `os.Exit()` should happen on the highest level possible, meaning `main.go`.